### PR TITLE
Enable lints

### PR DESCRIPTION
**Pull request by JustusFluegel**
Enables some more clippy lints

### CODE REVIEW

**Comment by JustusFluegel** [comment]
All failing lints are documented in #167

File: scripts/issue_creator.nu
Comment (NicMcPhee) [inline scripts/issue_creator.nu:1]: This was really cool! I like having both the master issue and all the sub-issues for each lint.

File: packages/ec-linear/examples/count_ones/main.rs
Comment (NicMcPhee) [inline packages/ec-linear/examples/count_ones/main.rs:1]: Do we want issues for the examples as well? I'm not sure about that, but I feel like we don't want to "publish" examples with things like `dbg!()` in them?

**Review by NicMcPhee — APPROVED** [review]
This is awesome – thanks!

**Pull request merged**